src/components/sections/About.section.tsx

```diff
@@ -19,11 +20,16 @@ const About: React.FC = () => {
     return (
         <div className="relative bg-gradient-to-b from-peachPuff to-brightUbe pb-10 pt-48">
             {/* MAIN SVG (Background) */}
-            <div className="relative w-screen">
+            <div className="relative w-screen object-cover">
```

Comment:
``` 
<div className="relative w-full">
``` 
w-screen causes overflow, using w-full instead of screen value

Reply:
Roger that :)